specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/shortTermRetentionPolicies.json

```diff
@@ -0,0 +1,653 @@
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the policy.",
+            "schema": {
+              "$ref": "#/definitions/ShortTermRetentionPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidRestorableDroppedDatabaseDeletionDate - The restorable dropped database deletion date given is invalid\n\n * 400 InvalidRestorableDroppedDatabaseId - Invalid restorable dropped database identifier\n\n * 400 InvalidParameterValue - An invalid value was given to a parameter.\n\n * 400 InvalidBackupRetentionPeriod - The retention days of {0} is not a valid configuration. Valid backup retention in days must be between {1} and {2}\n\n * 403 UpdateShortTermRetentionFeatureNotSupportedForEdition - This feature is not available for the selected database's edition {0}.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found.\n\n * 404 CannotFindObject - Cannot find the object because it does not exist or you do not have permissions\n\n * 404 SourceDatabaseNotFound - The source database does not exist.\n\n * 429 SubscriptionTooManyCreateUpdateRequests - Requests beyond max requests that can be processed by available resources.\n\n * 429 SubscriptionTooManyRequests - Requests beyond max requests that can be processed by available resources.\n\n * 503 TooManyRequests - Requests beyond max requests that can be processed by available resources.\n\n * 504 RequestTimeout - Service request exceeded the allowed timeout."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update the short term retention policy for the database.": {
+            "$ref": "./examples/UpdateShortTermRetentionPolicy.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ShortTermRetentionPolicies"
+        ],
+        "description": "Updates a database's short term retention policy.",
+        "operationId": "ShortTermRetentionPolicies_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "policyName",
+            "in": "path",
+            "description": "The policy name. Should always be \"default\".",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "ShortTermRetentionPolicyName",
+              "modelAsString": true
+            }
+          },

```

Comment:
Seems `policyName` is used in most of the operations, should we move this to parameter section?

Reply:
policyName will always be "default". I'd like to keep it consistent with what we see in [Long Term Retention](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json)

Reply:
The benefit would be the same that we get from having a common definition of resource group name or subscription parameter. Out of 8 operations in this swagger, 6 of them uses policyName param, if you centralize this definition then swagger looks more clean, less lines and is a good pattern to follow in general.

In Long Term Rentetion swagger, out of 8 only two uses policyName, so when reviewed it I didn't ask to move definition as it was not saving much.

Reply:
had an offline chat with @dealaus, swagger is not manually authored, SQL team will discuss internally to see whether this can be accommodated in the swagger gen tool. For now this is not a must fix for this PR.